### ISSUE TITLE
It seems `is_active` method does not exist anymore

### ISSUE THREAD

**Issue by Kerosene2000** [HeroesGrave/ecs-rs#33]
I think you recently updates the library code but not the guide one 😄 
https://github.com/HeroesGrave/ecs-rs/blob/master/doc/tutorial.md#making-a-system-passive

**Comment by HeroesGrave** [comment]
Yeah. I've been busy lately and have only managed to push updates that I needed for my current project. Documentation is really needed but I just don't have the time or energy to do it.

I'm actually in the process of rewriting the entire library to be simpler, more performant, and support multi-threading. It breaks enough things that I don't want to just abandon current users but at the same time it's hard to maintain the current library in terms of documentation, etc.

Instead of overriding the `is_active()` method, all you do now is add your system to the 'passive' section of the `SystemManager`.

**Comment by Kerosene2000** [comment]
Ok thank you !
No problem guy, I am now `watching` the project and waiting for new issues about the new API you just talked !
I will use your `ecs-rs` crate for my own game too 😄
Thanks for your work !
